.github/workflows/ci.yml

```diff
@@ -36,16 +36,10 @@ jobs:
           python -VV
           python -m site
           python -m pip install --upgrade pip
-          # setuptools needed for 3.12+ because of https://github.com/mesonbuild/meson/issues/7702.
```

Comment:
You seem to have undone the hack we were doing here before. Any particular reason or was it accidental?

I think that explains the CI failure.

Reply:
I think you want to drop 9284e0df9670f4dc30e9d547d5ba33ad4c70111d from your branch.

Reply:
Sorry for the delay. It was accidental, of course. No idea why it ended up in my branch...

I'll remove it and force push.